### PR TITLE
Remove Old: JanDeDobbeleer.OhMyPosh version 3.180.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
null

### CODE REVIEW

File: manifests/k/Kingsoft/KDocs/3.2.3/Kingsoft.KDocs.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Kingsoft.KDocs
+PackageVersion: 3.2.3
```
Comment (OfficialEsco) [inline manifests/k/Kingsoft/KDocs/3.2.3/Kingsoft.KDocs.installer.yaml:5]:
```suggestion
PackageVersion: 3.0.4
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
@SpecterShell

**Comment by SpecterShell** [comment]
It seems that they are using a new method for partial update and they didn't roll out the new installer. No idea if they forget to do so. 🤷
I have pushed a new source I found on the download page. https://github.com/vedantmgoyal2009/winget-pkgs-automation/issues/119

The old source:
```JSON
{
  "version": "3.2.3",
  "forceVersions": [
    {
      "version": "2.5.0",
      "time": "1615910400000"
    }
  ],
  "changes": "- 优化：上传体验升级，上传过程中可以选择替换同名文件，上传失败时可以直接点击重试，文件上云操作更快速；\n- 优化：整体性能优化，使用更流畅。",
  "fname": "KDocs_1002_v3.0.4.exe",
  "url": "https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_1002_v3.0.4.exe",
  "asar": "https://qn.cache.wpscdn.cn/kdocs/apk/asar/KDocs_v3.2.3_4a5c87241c0560ab8df11ce4d182a610.gz",
  "slient": true,
  "ignoreChannel": "",
  "channelVersion": {
    "2004": {
      "version": "3.1.1",
      "changes": "- 新增：金山文档模板库，更关注协作场景的模板库，注重一站式解决方案；多人统计、项目管理、教育学习等场景无从手下的问题，在金山文档模板库都能轻松解决！告别东找西找，不再苦恼；\n- 新增：以模板发送，文件列表右键-分享模板给好友，他也可以使用模板创建文档，再不需要另存等繁琐的操作啦！\n- 新增：右键菜单点击新窗口打开，可以同时开着多个窗口，工作效率大大提升；\n- 优化：整体性能优化，使用更流畅。",
      "fname": "KDocs_2004_v3.1.1.exe",
      "url": "https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_2004_v3.1.1.exe"
    }
  },
  "platformVersion": {
    "win7": {
      "version": "3.2.3",
      "changes": "- 优化：上传体验升级，上传过程中可以选择替换同名文件，上传失败时可以直接点击重试，文件上云操作更快速；\n- 优化：整体性能优化，使用更流畅。",
      "fname": "KDocs_1002_v3.0.4.exe",
      "url": "https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_1002_v3.0.4.exe",
      "asar": "https://qn.cache.wpscdn.cn/kdocs/apk/asar/KDocs_v3.2.3_4a5c87241c0560ab8df11ce4d182a610.gz",
      "slient": true
    }
  }
}
```

**Comment by vedantmgoyal2009** [comment]
😂🤣👌😁👍

**Comment by vedantmgoyal2009** [comment]
Updated the source in the automation now.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**